config/moab.mk

```diff
@@ -0,0 +1,30 @@
+$(MOAB_BUILDDIR)/Makefile: $(MOAB_DIR)/CMakeLists.txt
```

Comment:
Your comment about MOAB CMake settings on slack reminded me -- MOAB doesn't set a default build type (Release, Debug, etc.), and can be really slow without compile-time optimization. I'd recommend setting `CMAKE_BUILD_TYPE` to Release here.

Reply:
Thanks, that's really good to know! I'll add release